pkg/rpm.bzl

```diff
@@ -25,8 +25,17 @@ def _pkg_rpm_impl(ctx):
     if ctx.attr.debug:
         args += ["--debug"]
 
-    if ctx.attr.rpmbuild_path:
-        args += ["--rpmbuild=" + ctx.attr.rpmbuild_path]
+    toolchain = ctx.toolchains["//toolchains:rpmbuild_toolchain_type"].rpmbuild
+    if not toolchain.label and not toolchain.path:
+        fail("RpmbuildInfo must specify specify one of label or path")
+    if toolchain.path:
+        args += ["--rpmbuild=" + toolchain.path]
+        if toolchain.label:
+            fail("RpmbuildInfo should not specify both label and path")
```

Comment:
Shouldn't this failure detection be done when the toolchain is created?

Reply:
Good point. Moved there. And added analysis tests for it.